test/testUtils.ts

```diff
@@ -51,6 +54,74 @@ export const generateTestAccessToken = async (id: number): Promise<string> => {
   );
 };
 
+export interface CreateProjectData {
```

Comment:
I was thinking we have a seed file called helpers.ts Maybe we can reuse that? or adapt it for test env?

Reply:
Not required for this PR but there are a bunch of models there

Reply:
great idea, I had forgotten about that, I think in the future we can use that, thanks for mentioning